com.adrienplazas.Metronome.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id": "com.adrienplazas.Metronome",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "metronome",
+    "finish-args" : [
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/metronome/cargo"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a",
+        "/lib/girepository-1.0",
+        "/share/gir-1.0",
+        "/bin/sassc"
+    ],
+    "modules": [
+        {
+            "name" : "libadwaita",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dintrospection=disabled",
+                "-Dtests=false",
+                "-Dexamples=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+                    "tag" : "1.0.0-alpha.2",
+                    "commit" : "f5932ab4250c8e709958c6e75a1a4941a5f0f386"
+                }
+            ],
+            "modules": [
+                {
+                    "name" : "libsass",
+                    "buildsystem" : "meson",
+                    "cleanup" : [
+                        "*",
```

Comment:
```suggestion
                        "*"
```

Reply:
/me shakes a fist in JSON's general direction.